.stylelintrc

```diff
@@ -1,6 +1,7 @@
 {
   "rules": {
-    "declaration-no-important": true
+    "declaration-no-important": true,
+    "selector-type-no-unknown": [true, { "ignoreTypes": ["/^-styled-mixin/"] }]
```

Comment:
Could you please explain why you added this rule?

Reply:
See https://github.com/BBC-News/psammead/pull/105#pullrequestreview-182717916